README.md

```diff
@@ -93,380 +93,54 @@
-following command:
-
-``` bash
-pip install ivy
-```
-
-or alternatively:
-
-``` bash
-python3 -m pip install ivy
-```
-
-### Docker
-
-If you prefer to use containers, we also have pre-built Docker images
-with all the supported frameworks and some relevant packages already
-installed, which you can pull from:
-
-``` bash
-docker pull unifyai/ivy:latest
-```
-
-If you are working on a GPU device, you can pull from:
-
-``` bash
-docker pull unifyai/ivy:latest-gpu
-```
 
-### Installing from source
-
-You can also install Ivy from source if you want to take advantage of
-the latest changes, but we can\'t ensure everything will work as
-expected. :sweat_smile:
-
-``` bash
-git clone https://github.com/unifyai/ivy.git
-cd ivy
-pip install --user -e .
-```
-
-or alternatively, for the last step:
-
-``` bash
-python3 -m pip install --user -e .
-```
-
-If you want to set up testing and various frameworks it\'s probably best
-to check out the [Contributing - Setting
-Up](https://unify.ai/docs/ivy/overview/contributing/setting_up.html#setting-up)
-page, where OS-specific and IDE-specific instructions and video
-tutorials to do so are available!
-
-### Using Ivy
-
-You can find quite a lot more examples in the corresponding section
-below, but using Ivy is as simple as:
-
-#### Multi-backend Support
-
-``` python
-import ivy
-import torch
-import jax
-
-ivy.set_backend("jax")
-
-x = jax.numpy.array([1, 2, 3])
-y = jax.numpy.array([3, 2, 1])
-z = ivy.add(x, y)
+------------------------------------------------------------------------
 
-ivy.set_backend('torch')
 
-x = torch.tensor([1, 2, 3])
-y = torch.tensor([3, 2, 1])
-z = ivy.add(x, y)
-```
+## 📚 Documentation
 
-#### Transpilation API
+You can find our extensive documentation from this [Ivy Docs page](https://unify.ai/docs/ivy/)  which includes: 
+- [Docs](https://unify.ai/docs/ivy/): the full extensive documentation 
+    - [Design](https://unify.ai/docs/ivy/overview/design.html): the design decision, architecture, and building blocks (layers and nodes)  of Ivy. 
+    - [Deep
+dive](https://unify.ai/docs/ivy/overview/deep_dive.html)explains our code base and how to contribute to a specific field. 
+    - [Background](https://unify.ai/docs/ivy/overview/background.html): This contextualizes the problem Ivy is trying to solve through
+        - the current [ML
+Explosion](https://unify.ai/docs/ivy/overview/background/ml_explosion.html#ml-explosion),, 
+        - explaining both why is important [to solve this
+problem](https://unify.ai/docs/ivy/overview/background/why_unify.html#why-unify) and 
+        - how we adhere to existing [standards](https://unify.ai/docs/ivy/overview/background/standardization.html#standardization) to make this happen.
+    - [Related
+Work](https://unify.ai/docs/ivy/overview/related_work.html)   which paints a clear picture of the role Ivy plays in the ML stack, comparing it to other existing solutions in terms of functionalities and level.
```

Comment:
Would be better to have motivation -> related work -> design -> deep dive (from more general to more in-depth)